dev-docs/module-rules.md

```diff
@@ -40,65 +40,77 @@ Bid adapters, user ID adapters, analytics adapters, and other modules will be re
         - If run-time code is not needed for the auction, it should support a deferred load. E.g. User ID modules.
-        - The file loaded must be locked to a particular version so that any upgrades must be done via Prebid pull request.
-        - A prominent disclosure must be made about the loading of the file and why they need the file, and a size warning provided if it’s greater than 10KB.
-    1. If a functionality is supported by Prebid core or an existing module, Prebid modules must prefer the Prebid version of that functionality rather than an externally coded version.
-    1. The build-time or run-time loading of required or optional external files must be disclosed. (See the disclosure section below.)
-    1. Modules must not import other Prebid modules at build time.
-1. Modules must not(*)  add any pixel, iframe, cookie or local storage directly onto the page. Rather, they must use wrapper-provided mechanisms for usersyncs, cookies, and local storage.
+        - The file loaded is locked to a particular version so that any upgrades must be done via Prebid pull request.
+        - A prominent disclosure is made about the loading of the file and why they need the file, and a size warning provided if it’s greater than 10KB.
+    1. If a functionality is supported by Prebid core or an existing module, modules must prefer the Prebid version of that functionality rather than an externally coded version.
+    1. The build-time or run-time loading of required or optional external files must be disclosed. (See the [disclosure](#disclosure) section below.)
 1. Modules must use the communication functions provided by the Prebid core (both Prebid.js and Prebid Server) for external communication.
-1. Prebid.js modules must not use the $$PREBID_GLOBAL$$ variable nor otherwise obtain a pointer to the global PBJS object. This preserves data integrity.
+1. Modules must not add any pixel, iframe, cookie or local storage directly onto the page. Rather, they must use wrapper-provided mechanisms for usersyncs, cookies, and local storage.
 1. All modules must support the creation of multiple concurrent instances. This means, for example, they cannot rely on mutable global variables.
-    - Writing to the global window object must be non-intrusive, disclosed, and done in a way that supports multiple instances.
-1. All modules and any external code references must disclose their support of privacy regulations such as GDPR, CCPA, COPPA, etc. (See the disclosure section below.)
+    - PBJS: Writing to the global window object must be non-intrusive, disclosed, and done in a way that supports multiple instances.
+1. Modules must not include polyfills or override standard or Prebid JavaScript functions.
+1. Endpoint domain names cannot be fully variable. At some point, they won't be able contain any variables at all. e.g. $PARAM1.example.com.
+1. In order to be listed on the Prebid.org website, a module must be in a Prebid open source repository.
```

Comment:
Nitpick, you can omit "in order" from the start of the sentence and start with "To be listed..." without loosing meaning.